facebook_connector.py

```diff
@@ -0,0 +1,101 @@
+"""
+
+Author: Nigel Schuster <nigel schusters at gmail dot com>
+
+This module provides a simple abstraction for accessing the Facebook chat data
+
+"""
+
+from __future__ import print_function
+
+from bs4 import BeautifulSoup
+from fbchat_archive_parser.parser import MessageHtmlParser
+import fnmatch
+import os
+import pandas as pd
+import re
+import warnings
+
+_messages_file = None
+_fb_id_pattern = re.compile(r"(\d+)@facebook\.com")
+
+
+def initialize(dump_location="."):
+    """
+        Initializes the access to the Facebook messages file by asserting its location
+
+        Args:
+            dump_location: path to the messages.htm file
+    """
+    global _messages_file
+    fb_message_filename = "messages.htm"
+    _messages_file = os.path.join(dump_location, fb_message_filename)
+    if not os.path.isfile(_messages_file):
+        print("Please specify the location of the file in the function call".format(
```

Comment:
Update to refer to the dump.  Also you call `format()` but there's no part of this string that will be filled out.  Perhaps elaborate and say attempted to search for file at `<location>`